streamz/tests/test_core.py

```diff
@@ -199,6 +210,21 @@ def test_sliding_window_ref_counts():
         r_prev = r
 
 
+def test_sliding_window_metadata():
+    source = Stream()
+    stream = source.sliding_window(2)
+
+    L = []
+    sink = sink_metadata_to_list(stream, L)
+    stream.connect(sink)
```

Comment:
Ordinary `sink_to_list` doesn't need to be primed with a list, or have the sink object assigned and connected. This should probably do the same (I realise it's only for testing).